docs/user/custom-resources/apirule/04-60-apirule-v1beta2-custom-resource.md

```diff
@@ -60,11 +60,10 @@ This table lists all parameters of APIRule `v1beta2` CRD together with their des
 When you fetch an existing APIRule CR, the system adds the **status** section which describes the status of the
```

Comment:
This is not correct, since it mentions Virtual Service, we should just remove it.